functionnal_tests/cypress/integration/generic_frontend/operation_tinymce.js

```diff
@@ -86,8 +86,7 @@ describe('TinyMce text editor', function () {
 
       it('it should not leave any span after saving the content', function () {
         cy.waitForTinyMCELoaded().then(() => {
-          cy.inputInTinyMCE('@')
-          cy.inputInTinyMCE('jo')
+          cy.inputInTinyMCE('@johndoe')
           cy.get('.html-document__editionmode__submit').click()
           cy.get('.html-document__contentpage__textnote__version').should('be.visible')
           cy.get('#autocomplete').should('be.not.visible')
```

Comment:
Here, l'autocomplete should not be visible or should not exist? (that already cause some "random" fails at cypress)

Reply:
I do not know, I just changed the input so that the username is valid (this was failing this test). In which case are you expecting a random behavior? Testing `be.not.visible` if the code behavior is to remove the `#autocomplete` element?

BTW you can see who changed the PR by viewing its commits :-)

Reply:
When I had problems with this command it was because `get` gave error (since the element didn't exist) and it didn't check `not.visible`, but sometimes it worked, so I can't tell you why it was something very random for me and worked when I changed it to `not.exist`.

(there is a lot of commits :laughing: )